src/nl/hannahsten/texifyidea/psi/BibtexEntryUtil.kt

```diff
@@ -21,8 +21,13 @@ fun getReferences(element: BibtexEntry): Array<PsiReference> {
         // doesn't know which text to underline). So we have to return a list of
         // WebReference(element, text range) instead.
         val contentText = element.entryContent?.text ?: return emptyArray()
-        val rangesInParent = urls.map {
-            TextRange.from(contentText.indexOf(it), it.length)
+        val rangesInParent = urls.mapNotNull {
+            if (it in contentText) {
+                null
+            }
+            else {
+                TextRange.from(contentText.indexOf(it), it.length)
+            }
```

Comment:
This doesn't make sense. This always returns an empty array, since `getTagContent()` will only search for the given tag in the content text of the given element. So `it in contentText` will always return `true`. As a result, the links in BibTeX are not clickable (anymore). Maybe we should check if the returned text range is in the range of the content text instead?

Reply:
Indeed that didn't make sense, that should have been `it !in contentText` of course.

> So it in contentText will always return true

Nope, sorry. See the second error I put in #1246

> Maybe we should check if the returned text range is in the range of the content text instead?

I think if `it in contentText` then `0 <= contentText.indexOf(it)` and `it.length <= contentText.length` so it should effectively be the same thing.

Reply:
> Nope, sorry. See the second error I put in #1246

I see. But I don't understand when that happens. Any example that triggers that?

Reply:
No, I couldn't reproduce the error